Documentation/contributing/testing/ci.rst

```diff
@@ -185,12 +185,12 @@ them all at once:
 +==================+==========================+
 | master           | /test                    |
 +------------------+--------------------------+
+| v1.11            | /test-backport-1.11      |
```

Comment:
Do we need to take any action to ensure this is set up correctly? CC @nbusseneau

Reply:
(Particularly on the jenkins side; looks like this PR already handles the GHA side)

Reply:
Indeed, there a bunch of actions required on Jenkins' side to add the new trigger, and then we should update the CI Matrix. I will also phase out 1.8 from Jenkins since we have removed it from `master` with this PR.

Reply:
The `.github/maintainers-little-helper.yaml` file will also need to be udpated on branch 1.11 with the new Jenkins 1.11 jobs.

Reply:
Jenkins has been updated:

- 1.11 automated jobs available at https://jenkins.cilium.io/view/Cilium-v1.11/
- `/test-backport-1.11` available

See [CI Matrix](https://docs.google.com/spreadsheets/d/1TThkqvVZxaqLR-Ela4ZrcJ0lrTJByCqrbdCjnI32_X0/edit) for more information.

GHA-side changes are tracked in #17986, will review there.